src/main/java/survey/Question.java

```diff
@@ -79,4 +95,31 @@ public String getQuestion() {
     public void setQuestion(String question) {
         this.question = question;
     }
+
+    /**
+     * Get the minimum boundary for a question of type number_question
+     * @return String
+     */
+    public String getMin() {
```

Comment:
Not a necessary change, but IMO we don't need javadocs for getters/setters UNLESS they do something outside of the normal getter/setter expected behaviour.